### DISCUSSION TITLE
New bug for broken GitHub App auth?

### DISCUSSION THREAD

**Discussion by joebowbeer** [argoproj/argo-cd#6194] [Q&A] [answered]
About the broken GitHub App auth in Argo CD: https://github.com/argoproj/argo-cd/issues/3086

Should I create a new bug for this or continue to watch the old one? 

Since the old bug was presumed fixed by some, and the conversation thread is long, should I create a new bug?

**Comment by jgwest** [comment, marked as answer]
Hi @joebowbeer, I'd recommend opening a new bug: since the existing functionality has been merged and is now in the project, and since (based on comments) it appears to be a semi-recent regression, a new bug makes sense to me.

**Reply by joebowbeer** [reply]
Thanks! Bug: #6196 PR: #6197
